codes/scripts/post_install.py

```diff
@@ -0,0 +1,140 @@
+import filecmp
+import os
+import shutil
+import sys
+import sysconfig
+import subprocess
+from pathlib import Path
+import requests
+import zipfile
+if sys.version_info < (3, 8):
+    import importlib_metadata
+else:
+    import importlib.metadata as importlib_metadata
+
+req_file = os.path.join(os.getcwd(), "requirements.txt")
```

Comment:
I'm probably also going to clean up this file and remove redundant lines like these, unless you have objections